kernel-default.properties

```diff
@@ -442,8 +436,8 @@ mosip.service.end-points={cipher}dd5737cb38b0d354e925a031d1662cae45feb891135577d
 
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 
-##validity time of mosip-signed certificates
-
+mosip.kernel.masterdata.template_idauthentication_event=MASTERDATA_IDAUTHENTICATION_TEMPLATES
+mosip.kernel.masterdata.title_event=MASTERDATA_TITLES
```

Comment:
Is this java21 changes?

Reply:
it's not part of java21 but it's required for publish new title that's add part of masterdata